recipes/pymumps/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymumps" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/santiagoropb/pymumps/archive/v{{ version }}.tar.gz
+  sha256: dddfc13eb6af56a9bcd8e42fd42c3293dc50fd155fe3847c3dd84a403edba8ba
+
+build:
+  number: 1000
+  script: python setup.py install --include-dirs $BUILD_PREFIX/include --library-dirs $BUILD_PREFIX/lib
```

Comment:
Typically this is done with pip instead.

Reply:
I am linking against the mumps libraries that are installed with conda. Therefore, I am not sure how can this be done with pip since the built needs to know where those libraries are. Since this is working I would like to keep it as is. I keep this in mind for future recipes where compilation with cython is not required.

Reply:
I have look at other recipes in the feedstock and this seems to be reasonable. e.g https://github.com/conda-forge/pyomo-feedstock/blob/master/recipe/meta.yaml